cabot/metricsapp/models/base.py

```diff
@@ -82,33 +83,57 @@ def update_url(self):
                   'threshold before an alert is triggered. Applies to both warning '
                   'and high-alert thresholds.',
     )
+    on_empty_series = models.CharField(
+        choices=defs.ON_EMPTY_SERIES_OPTIONS,
+        default=defs.ON_EMPTY_SERIES_FILL_ZERO,
+        max_length=16,
+        help_text='Action to take if the series is empty. Options are: pass, warn, or fail immediately, '
+                  'or insert a single data point with value zero (default, for backwards compatibility).'
```

Comment:
I would remove the `for backwards compatibility` - it won't impact new users, and you can call this out in the release notes email.

Reply:
sounds good, i'll remove it